package.json

```diff
@@ -70,8 +71,8 @@
     "babel-plugin-transform-class-properties": "6.11.5",
     "babel-plugin-transform-es2015-modules-commonjs": "6.14.0",
     "babel-plugin-transform-flow-strip-types": "6.14.0",
-    "babel-preset-es2015": "6.14.0",
     "babel-plugin-transform-runtime": "6.15.0",
+    "babel-preset-es2015": "6.14.0",
```

Comment:
I didn't do this, but I assume npm auto-sorted the dependencies when I installed `nsp` as a devDep.